docs/sidebar.ts

```diff
@@ -44,6 +44,7 @@ export const sidebarItems: ISidebarSection[] = [
       { href: "/docs/providers/go", text: "Go" },
       { href: "/docs/providers/haskell", text: "Haskell" },
       { href: "/docs/providers/java", text: "Java" },
+      { href: "/docs/providers/lunatic", "text: "Lunatic" },
```

Comment:
```suggestion
      { href: "/docs/providers/lunatic", text: "Lunatic" },
```

Caught by CI in https://github.com/railwayapp/nixpacks/actions/runs/4921867678/jobs/8824973626?pr=873.

Reply:
Thanks good catch :)